3p-integrations/gcp/vertex_MaaS/Vertex_JSON_mode_for_Llama_4.ipynb

```diff
@@ -0,0 +1,160 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "provenance": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "code",
```

Comment:
Please add context and what this notebook is showing , similar to the tool calling doc